includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -1969,11 +1969,32 @@ protected function build_display_items( $itemized_display_items = false ) {
 			];
 		}
 
+		$calculated_total = apply_filters_deprecated(
+			'woocommerce_stripe_calculated_total',
+			[ WC_Stripe_Helper::get_stripe_amount( $order_total ), $order_total, WC()->cart ],
```

Comment:
Another situation where we could choose to initialise the value before we call the deprecated filter.
```suggestion
		$calculated_total = WC_Stripe_Helper::get_stripe_amount( $order_total );
		$calculated_total = apply_filters_deprecated(
			'woocommerce_stripe_calculated_total',
			[ $calculated_total, $order_total, WC()->cart ],
```